lib/record_store/record/caa.rb

```diff
@@ -5,7 +5,11 @@ class Record::CAA < Record
     LABEL_REGEX = '[a-z0-9](?:-*[a-z0-9])*'
     DOMAIN_REGEX = /\A#{LABEL_REGEX}(?:\.#{LABEL_REGEX})\z/i
 
-    validates :flags, numericality: { only_integer: true, greater_than_or_equal_to: 0, less_than_or_equal_to: 255 }, presence: true
+    validates :flags, numericality:
```

Comment:
Would suggest to move `, presence: true` before the `numericality:` key, and as above, keep the `{` on the same line as `numericality:`